docs/API.AMQP.md

```diff
@@ -261,7 +269,44 @@ How subscription works:
   },
   metadata: {
     // query metadata
-  }
+  },
+
+  /*
+  The "scope" argument filters document modifications when a change has been
+  detected in the database.
+  You may receive notifications only when documents enter your
+  scope (scope: 'in'), when documents leave it (scope: 'out'), or
```

Comment:
for (scope: 'in'), isn't it rather "when documents enter or stay in your scope" ?